### DISCUSSION TITLE
Sorobanathon: First Light 🔭 has Officially Ended!

### DISCUSSION THREAD

**Discussion by achallagundla** [stellar/sorobanathon#175] [Announcements]
WOW! What a fantastic set of submissions, thank you all for participating in _Sorobanathon: First Light 🔭_. 
We look forward to seeing you participate in many more events, challenges, and competitions around Soroban! 

Please be patient as we go through and review all of these amazing submissions, and get your rewards out. 
Planning to have it done by December 31st, 2022, at the latest. 

Wish you all a wonderful holiday season! 
As always you can reach out to me on Discord @anuxhya.

**Comment by Stellarhamza** [comment]
Thank you for such a amazing experince!

**Comment by stellarjohn** [comment]
Thanks to everyone that submitted!

**Comment by wildework** [comment]
This was one of the best onboarding experiences to new tech I've seen. Congrats!

**Comment by hanseartic** [comment]
Thank you for this really well-thought event. It really was fun and was a good incentive to get hands-on in a timely manner.

But what I liked even more about it was then welcoming way you set it up; it was open to a broad range of contributions - not 'only' tech-/dev-focused but embracing every kind of contribution 😍.
